doc/source/guidelines/PyMAPDL_WSL_Guide.rst

```diff
@@ -0,0 +1,423 @@
+
+
+In case of Ubuntu, please follow the specific instructions in `Running MAPDL: Ubuntu <https://mapdldocs.pyansys.com/getting_started/running_mapdl.html#ubuntu/>`_ .
+
+
+Install ANSYS
+--------------
+
+To install ANSYS in WSL Linux, you need to follow the next steps.
+
+1. Download ANSYS Structures image from customer portal (`Current Release <https://download.ansys.com/Current%20Release>`_). 
+   If you are using windows to download it, you can later copy it from it to WSL (recommended).
+
+2. Decompress the source code file (tar.gz) by using 
+
+   .. code:: bash
+   
+       tar xvzf STRUCTURES_2021R2_LINX64.tgz
+
+
+3. Install ANSYS by going into the uncompressed folder and type:
+
+   .. code:: bash
+   
+       ./INSTALL -silent -install_dir /usr/ansys_inc/ -mechapdl
+
+   where: 
+
+   - ``-silent`` : Initiates a silent installation (No GUI).
+
+   - ``-install_dir /path/`` : Specifies the directory to which the product or license
+     manager is to be installed. 
+     If you want to install to the default location, you can omit the ``-install_dir`` argument. 
+     The default location is ``/ansys_inc`` if the symbolic link is set; otherwise, it will default to ``/usr/ansys_inc``.
+
+   - ``-<product_flag>`` : Specifies one or more products to install specific products. 
+     If you omit the -product_flag argument, all products will be installed. 
+     See the list of valid product_flags in the Chapter 6 in *ANSYS Inc. Installation Guides* PDF. 
+     In this case, only mechanical (`-mechapdl`) is needed.
+
+You could install it directly in ``/ansys_inc`` if you wish, or in ``/usr/ansys_inc`` and then create a symbolic link using:
+
+.. code:: bash
+
+    ln -s /usr/ansys_inc /ansys_inc
+
+By default, PyMAPDL expects ANSYS executable being in ``/usr/ansys_inc``, so whether you install it there or not, it is recommended you link that directory, using symbolic link, to your ANSYS installation directory (``/*/ansys_inc``).
+
+
+Post installation setup
+========================
+
+Opening ports
+--------------
+
+**Theory:** 
+You should open the ports ``1055`` and ``2325`` for the license server communication in `Windows Firewall Advanced`.
+You can see the steps in `How to open port in Windows 10 Firewall <https://answers.microsoft.com/en-us/windows/forum/all/how-to-open-port-in-windows-10-firewall/f38f67c8-23e8-459d-9552-c1b94cca579a/>`_ . 
+
+**Reality:**
+This works if you want to run a docker image using WSL Linux image to host that docker image.
+The docker image will successfully communicate with the Windows License Server using these ports if using ``'-p'`` flag when running the docker image and if having those ports open.
+See `Running ANSYS on a local docker`_ .
+
+
+If you wish to run ANSYS in the CentOS7 image and use the Windows License Server, opening the ports might not work properly, since Windows firewall seems to block all traffic coming from WSL. 
```

Comment:
```suggestion
If you want to run MAPDL in the CentOS7 image and use the Windows License Server, opening the ports might not work properly because the Windows firewall seems to block all traffic coming from WSL. 
```